lib/flutter_map.dart

```diff
@@ -90,13 +92,22 @@ typedef void TapCallback(LatLng point);
 typedef void LongPressCallback(LatLng point);
 typedef void PositionCallback(MapPosition position, bool hasGesture);
 
+/// Allows you to provide your map's starting properties for [zoom], [rotation]
+/// and [center].
+/// Zoom, pan boundary and interactivity constraints can be specified here too.
+///
+/// Callbacks for [onTap], [onLongPress] and [onPositionChanged] can be
+/// registered here.
+///
+/// Through [crs] the Coordinate Reference System can be
+/// defined, it defaults to [Epsg3857].
 class MapOptions {
   final Crs crs;
   final double zoom;
   final double rotation;
   final double minZoom;
   final double maxZoom;
-  final bool debug;
+  final bool debug; // TODO no usage outside of constructor. Marked for removal?
```

Comment:
yes we can either remove this or mark it as `@deprecated`

Reply:
i was feeling a little pedantic and deprecated it. Maybe it could be removed at the first API change for MapOptions.